### PR TITLE
suit cosmetics

### PR DESCRIPTION
**Pull request by duncathan**
- [x] Player 2
- [x] Player 3
- [x] Player 4

image
image
image

### CODE REVIEW

**Comment by duncathan** [comment]
Player 2
image
image
image

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

File: src/open_prime_rando/echoes/schema.json [hunk 1]
```diff
@@ -67,6 +67,35 @@
...
+                        }
+                    },
+                    "additionalProperties": false,
+                    "default": {}
+                }
+            },
+            "additionalProperties": false,
+            "default": {
+                "suits": {
+                    "varia": "player2",
```
[the remark this answers is not in the document]
Comment (duncathan) [reply src/open_prime_rando/echoes/schema.json:94]: yes

**Comment by duncathan** [comment]
Player 3
image
image
image

**Comment by duncathan** [comment]
Player 4
image
image
image

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**